src/client/styles/MainPage.css

```diff
@@ -35,15 +35,15 @@ body {
     justify-content: center;
     flex-wrap: wrap;
     width: 100vw;
-    font-size: 1.4rem;
+    font-size: 3vh;

```

Comment:
What is the reason for using viewport height instead of rem?

Reply:
rem is relative to root pixelsize. Whereas using viewport height is relative to the window.innerHeight.
So either there should be a font-size relative to viewport in body (which there haven't been so far) and readjust rem font-size in the rest of the css file.